semantic-python/src/Language/Python/Core.hs

```diff
@@ -281,17 +281,17 @@ instance Compile Py.List
 instance Compile Py.ListComprehension
 
 instance Compile Py.Module where
-  compile it@Py.Module { Py.extraChildren = stmts } _cc = do
+  compile it@Py.Module { Py.extraChildren = stmts } _cc =
```

Comment:
We could start to use the `_cc` parameter here, but it’s `pure` at this point anyway, so I think it doesn’t matter much.

Reply:
I think it probably matters, since it would result in modules being compiled to `#record {…} ; ()` (or whatever the syntax for `unit` ends up being)? So in fact we _want_ to ignore it 👍

(Not certain about that, tho; I’d trust the evidence of your tests over my reasoning.)